docs/app/templates/getting-started/assets.hbs

```diff
@@ -27,21 +27,18 @@
 <h2 id="importing-css-assets">Importing static CSS Assets</h2>
 
 <p>
-  When not using a CSS preprocessor (see below) ember-bootstrap will
-  automatically import the static Bootstrap CSS into your ember-cli build. In
-  situations where you prefer to use another strategy for importing Bootstrap
-  CSS, you can opt out of CSS import by setting the
-  <code>importBootstrapCSS</code>
-  option to false in your
-  <code>ember-cli-build.js</code>:
+  When not using a CSS preprocessor (see below) you will need to import the
+  static Bootstrap CSS into your ember-cli build:
 </p>
 
 <div>
+  {{! prettier-ignore }}
   <pre class="highlight"><code>//your-bootstrap-app/ember-cli-build.js
-      module.exports = function(defaults) { let app = new EmberApp(defaults, {
-      'ember-bootstrap': { 'importBootstrapCSS': false } }); return
-      app.toTree(); };
-    </code></pre>
+module.exports = function(defaults) {
+  let app = new EmberApp(defaults, { /* Configuration */ });
+  app.import('node_modules/bootstrap/dist/css/bootstrap.css');
```

Comment:
I don't think we should suggest folks to use `app.import()` as that is a legacy EmberCLI API. I would rather suggest to use the "new" way using the ember-auto-import/webpack bundling pipeline, which is also how things would work natively under Embroider. Which is to import the CSS in JS code and let webpack handle the rest.

Haven't tried this out, but I think if you add this to your app's `app.{j,t}s`, then it _should_ work (we need to confirm!):

```
import 'bootstrap/dist/css/bootstrap.css';
```

Reply:
Aha, I was not aware of that. Luckily your suggestion seems to work like a charm both in a classic and Embroider build.

Reply:
Ember CLI docs document `app.import()` to pull in third-party CSS: https://guides.emberjs.com/release/addons-and-dependencies/#toc_css I think we should point consumers to that official documentation.

I'm okay with an additional note that this may change in future. And different patterns available today already.

I see the main intent for deprecating this (beside unblocking v2 conversion) in getting rid of the responsibility. I fear we would somehow keep them if recommending an unofficial CSS import pattern.

Reply:
> Ember CLI docs document `app.import()` to pull in third-party CSS: https://guides.emberjs.com/release/addons-and-dependencies/#toc_css I think we should point consumers to that official documentation.
> 
> I'm okay with an additional note that this may change in future. And different patterns available today already.
> 
> I see the main intent for deprecating this (beside unblocking v2 conversion) in getting rid of the responsibility. I fear we would somehow keep them if recommending an unofficial CSS import pattern.

P.S.: Upgrading Ember CLI docs is likely best mid- to long-term solution here. 😅